sklearn/datasets/samples_generator.py

```diff
@@ -696,23 +697,29 @@ def make_moons(n_samples=100, shuffle=True, noise=None, random_state=None):
     return X, y
 
 
-def make_blobs(n_samples=100, n_features=2, centers=3, cluster_std=1.0,
+def make_blobs(n_samples=100, n_features=2, centers=None, cluster_std=1.0,
                center_box=(-10.0, 10.0), shuffle=True, random_state=None):
     """Generate isotropic Gaussian blobs for clustering.
 
     Read more in the :ref:`User Guide <sample_generators>`.
 
     Parameters
     ----------
-    n_samples : int, optional (default=100)
-        The total number of points equally divided among clusters.
+    n_samples : int or array-like, optional (default=100)
+        If int, it is the the total number of points equally divided among
+        clusters.
+        If list, tuple or array, each element of the iterable indicates
```

Comment:
"array-like" and "sequence" are better than "list, tuple or array" and "iterable"

Reply:
Ok!